find-start-lms.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+RED="\x1B[31m"
+GREEN="\x1B[32m"
+NC="\x1B[0m"
+
+if [ -z ${DEVSTACK_WORKSPACE+x} ]; then DEVSTACK_WORKSPACE=$(pwd)/../../openedx; fi
```

Comment:
I think the default setup is to have the clone of the devstack repository be a sibling directory to commerce-coordinator. See [these devstack instructions](https://github.com/openedx/devstack#directions-to-setup-devstack).

Reply:
i was grouping by github org, heh... seemed more rational, will fix.

Reply:
Yes, as `if [ -z ${DEVSTACK_WORKSPACE+x} ]; then DEVSTACK_WORKSPACE=$(pwd)/..; fi` this is correct, right?